yt/utilities/lib/platform_dep.h

```diff
@@ -1,59 +1,9 @@
 #include <math.h>
 #ifdef MS_WIN32
 #include "malloc.h"
+// note: the following implicitly requires _MSC_VER >= 1928 (VS 2019, 16.8)
 #include <float.h>
```

Comment:
can you link the documentation in a comment ?

Reply:
yes, the note applies to the `#include <float.h>` as well, though I'm honestly not positive on what the minimum version of VS would be since it depends on the C standard used for compilation and what we needed from `float.h` to begin with? either way, I'll update the comment here.